src/styles/components/landing_page.scss

```diff
@@ -5,11 +5,11 @@
  */
 
 // Constants
-$FONT_SIZE_LINK: 48px;
+@import "../constants.scss";
```

Comment:
You might not need this since the `main.scss` already imports everything together...check for me though